packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.controller.js

```diff
@@ -0,0 +1,5 @@
+export default class IpByoipDisclaimer {
+  cancel() {
+    return this.goBack();
```

Comment:
If no tracking to plan better to use `goBack` in the HTML template.

Reply:
Have removed controller for now